includes/transformer/class-base.php

```diff
@@ -162,28 +165,40 @@ public function set_content_visibility( $content_visibility ) {
 	 * @return Base_Object The ActivityPub Object.
 	 */
 	protected function set_audience( $activity_object ) {
-		$public = 'https://www.w3.org/ns/activitystreams#Public';
-		$actor  = Actors::get_by_resource( $this->get_attributed_to() );
-		if ( ! $actor || is_wp_error( $actor ) ) {
-			$followers = null;
-		} else {
+		$public     = 'https://www.w3.org/ns/activitystreams#Public';
+		$followers  = null;
+		$replied_to = null;
+
+		$actor = Actors::get_by_resource( $this->get_attributed_to() );
+		if ( $actor && ! \is_wp_error( $actor ) ) {
 			$followers = $actor->get_followers();
 		}
+
 		$mentions = array_values( $this->get_mentions() );
 
+		if ( \method_exists( $this, 'get_in_reply_to' ) && $this->get_in_reply_to() ) {
+			$object = Http::get_remote_object( $this->get_in_reply_to() );
+			if ( $object && ! \is_wp_error( $object ) && isset( $object['attributedTo'] ) ) {
```

Comment:
```suggestion
			if ( isset( $object['attributedTo'] ) ) {
```
We should be able to shorten it to the `isset()` check. It'll return false for WP_Errors and falsy values.

Reply:
because WP_Error is an object, it fails when you try to handle it like an array... should be the same for other "falsy" values.